_unittest/test_20_HFSS.py

```diff
@@ -1624,3 +1624,30 @@ def test_67_transient_composite(self, add_app):
         aedtapp.solution_type = "Transient Composite"
         assert aedtapp.solution_type == "Transient Composite"
         aedtapp.close_project(save_project=False)
+
+    @pytest.mark.skipif(config["NonGraphical"], reason="Test fails on build machine")
+    def test_68_import_gds_3d(self):
+        self.aedtapp.insert_design("gds_import_H3D")
+        gds_file = os.path.join(local_path, "../_unittest/example_models", "cad", "GDS", "gds1.gds")
+        # list of gds number
+        gds_number = [7, 9]
```

Comment:
@amichel0205 it's a minor comment but for me you could directly pass these inputs (as well as importmethod) directly to the method without storing them in a vr.

Reply:
in reality the list can be much longer like this:
gds_number = [3, 6, 7, 8, 9, 10, 11, 30, 31, 32, 33, 34, 35]
elevation = [375.355, 375.575, 376.055, 376.655, 377.455, 378.055, 378.855, 379.455, 380.255, 380.855, 381.655, 
             382.405, 383.205]
thickness = [0.2, 0.5, 0.6, 0.8, 0.6, 0.8, 0.6, 0.8, 0.6, 0.8, 0.75, 0.8, 3]

Reply:
@amichel0205 in the test I meant, you can simply pass the list without storing in variables.
It's better to do it in this way:
assert self.aedtapp.import_gds_3d(gds_file_path=gds_file, gds_number_list=gds_number [...])
so you specify the input name.

Reply:
It seems 
@codecov codecov/patch Failing after 1s — 7.40% of diff hit (target 85.00%)

is failing after 1s do you have any idea why